app/views/publishers/publisher_preferences/_form.html.slim

```diff
@@ -13,19 +13,23 @@
       text_method: :name,
       hint_method: :address
 
-  = f.govuk_check_boxes_fieldset :school_ids_fieldset,
-    legend: { text: t(".schools_out", organisation: current_organisation.name) },
-    hint: { text: t(".schools_out_hint_html", email: govuk_mail_to(t("help.email"), t("help.email"))) },
-    classes: "checkbox-label__bold govuk-!-margin-top-5" do
+  - if current_organisation.schools_outside_local_authority.not_closed.any?
+    = f.govuk_check_boxes_fieldset :school_ids_fieldset,
+      legend: { text: t(".schools_out", organisation: current_organisation.name) },
+      hint: { text: t(".schools_out_hint_html", email: govuk_mail_to(t("help.email"), t("help.email"))) },
+      classes: "checkbox-label__bold govuk-!-margin-top-5" do
 
-    = render SearchableCollectionComponent.new form: f,
-      input_type: :checkbox,
-      label_text: "search schools outside local authority",
-      threshold: 10,
-      attribute_name: :school_ids,
-      collection: current_organisation.schools_outside_local_authority.not_closed.order(:name),
-      value_method: :id,
-      text_method: :name,
-      hint_method: :address
+      = render SearchableCollectionComponent.new form: f,
+        input_type: :checkbox,
+        label_text: "search schools outside local authority",
+        threshold: 10,
+        attribute_name: :school_ids,
+        collection: current_organisation.schools_outside_local_authority.not_closed.order(:name),
+        value_method: :id,
+        text_method: :name,
+        hint_method: :address
+  - else
+    h3.govuk-heading-s = t(".schools_out", organisation: current_organisation.name)
+    p.govuk-body = t(".schools_out_hint_html", email: govuk_mail_to(t("help.email"), t("help.email")))
```

Comment:
I think this hint text still should be shown to users who have some `schools_outside_local_authority`, because they might need to add another.

Reply:
In order to add schools outside the local authority, they have to send an email. will add a screenshot of how it was before so maybe it is more clear

Reply:
Yes. I'm saying, we should always display that information, even if they already have an outside school. These changes will hide those instructions if they have one outside school already, so if they want to add a second outside school, it will not be obvious how to.

Reply:
that text is show if they there are schools outside local authorities:
```
= f.govuk_check_boxes_fieldset :school_ids_fieldset,
      legend: { text: t(".schools_out", organisation: current_organisation.name) },
      hint: { text: t(".schools_out_hint_html", email: govuk_mail_to(t("help.email"), t("help.email"))) },
```
and when there are no schools outside local authority:
```
h3.govuk-heading-s = t(".schools_out", organisation: current_organisation.name)
    p.govuk-body = t(".schools_out_hint_html", email: govuk_mail_to(t("help.email"), t("help.email")))
```